package.json

```diff
@@ -0,0 +1,78 @@
+{
+  "name": "bfx-cli",
+  "version": "1.0.0",
+  "description": "Bitfinex CLI Trading Interface",
+  "engines": {
+    "node": ">=8.3.0"
+  },
+  "bin": {
+    "bfx-cli": "index.js"
+  },
+  "main": "index.js",
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm test"
+    }
+  },
+  "scripts": {
+    "postinstall": "npm run gen-salt",
+    "gen-salt": "node scripts/gen_salt.js",
+    "lint": "eslint lib/ index.js",
+    "test": "npm run lint && npm run unit",
+    "unit": "NODE_ENV=test nyc --check-coverage --lines 60 --branches 40 --functions 60 --statements 60 --reporter=lcov --reporter=html mocha -b --recursive"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/f3rno/bfx-cli.git"
```

Comment:
should we use https://github.com/bitfinexcom/bfx-cli.git instead ?

Reply:
Fixed, forgot about that, thanks